docs/source/releases/Beadledom40.rst

```diff
@@ -0,0 +1,30 @@
+.. _4.0:
+
+Beadledom 4.0
+=============
+
+To be released...
+
+New Features
+------------
+
+- TBD
+
+Migrating from Beadledom 3.0
+----------------------------
+
+Maven Module Changes
+~~~~~~~~~~~~~~~~~~~~
+The beadledom-swagger1 module was removed. This will require updating your poms that have a dependency
+on beadledom-swagger1.
```

Comment:
> and updating to beadedom-swagger2.

You may also want to link to the user guide for beadledom-swagger2, https://engineering.cerner.com/beadledom/3.4/docs/manual/swagger2.html

Reply:
9fa6f374